airbyte-webapp/src/components/JobItem/components/ResetStreamDetails.module.scss

```diff
@@ -0,0 +1,29 @@
+@use "../../../scss/colors";
+@use "../../../scss/variables";
+
+.textContainer {
+  margin: variables.$spacing-sm 0 0;
+  width: 100%;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  line-height: 20px;
+
+  &.open {
+    white-space: unset;
+  }
+}
+
+.text {
+  display: inline-block;
+  padding: 0 variables.$spacing-sm;
+  background: colors.$grey-50;
+  border: variables.$border-thin solid colors.$grey-100;
+  border-radius: variables.$border-radius-xs;
+  margin-right: variables.$spacing-md;
+  margin-bottom: variables.$spacing-sm;
+  white-space: nowrap;
+  font-weight: 400;
+  font-size: 12px;
```

Comment:
I think better keep it like this since, Text styles overrides line-height and we will need to use `!important` in this case, so I think better left as it is

Reply:
We can pass`line-height` in `<Text>` component via className  🤷🏻‍♂️
<img width="301" alt="image" src="https://user-images.githubusercontent.com/20929344/191855978-4e61e75d-a36b-4c32-bf2c-7bb45943c014.png">